hydroshare/settings.py

```diff
@@ -719,3 +719,8 @@
 
 # detect test mode to turn off some features
 TESTING = len(sys.argv) > 1 and sys.argv[1] == 'test'
+
+# HydroShare web services settings
+HSWS_URL = "http://10.37.155.109:8000/apps/hydroshare-web-services-manager/api"
+HSWS_API_TOKEN = "ee3844fc9814d2d23c9843e9a0717e58d0173a6f"
+HSWS_TIMEOUT = 20
```

Comment:
@kjlippold Looks to me at least these two geoserver web service specific settings, ```HSWS_URL``` and ```HSWS_API_TOKEN``` should go to local_settings.py rather than settings.py. At least HSWS_API_TOKEN needs to go to local_settings.py since it should not be exposed to outside world. I suggest to move them to local_settings.py. Since we don't check in local_settings.py in github now, you may put it in local_settings.cutover. If HSWS_API_TOKEN is sensitive info, you can just put a placeholder there, to be replaced with real values in a deployment environment.